src/math/Kconfig

```diff
@@ -31,6 +30,14 @@ config BINARY_LOGARITHM_FIXED
 	  with a short lookup table. (log2n) operates for a range of 32 bit width size
 	  i.e. 1 to 4294967295.
 
+config SQRT_FIXED
+	bool "Square Root functions"
+	default n
+	help
+	  Select this to enable sqrt_int() functions as 16 bit version
+	  to  calculate square root.square function having positive number
```

Comment:
Strange, checkpatch says:
```
WARNING: please write a paragraph that describes the config symbol fully
#76: FILE: src/math/Kconfig:33:
+config SQRT_FIXED
```
Formatting issue?

Reply:
don't know !! I tried inserting additional text as well but it didn't go away.